NitroxClient/Communication/Packets/Processors/CellEntitiesProcessor.cs

```diff
@@ -2,64 +2,42 @@
 using NitroxClient.GameLogic.Helper;
 using NitroxModel.DataStructures.Util;
 using NitroxModel.GameLogic;
-using NitroxModel.Logger;
+using NitroxModel.Helper;
 using NitroxModel.Packets;
-using System.Collections.Generic;
 using UnityEngine;
 
 namespace NitroxClient.Communication.Packets.Processors
 {
     class CellEntitiesProcessor : ClientPacketProcessor<CellEntities>
     {
-        private PacketSender packetSender;
-        private HashSet<string> alreadySpawnedGuids = new HashSet<string>();
-        
+        private readonly PacketSender packetSender;
+
```

Comment:
I'm not sure that this refactor will work because entities should unload when they become out of range.  If a chunk unloads and you receive this packet it will spawn another entity.